workspace/__lib__/xod/icons/bmp/patch.cpp

```diff
@@ -0,0 +1,16 @@
+
+struct Bmp {
+    uint8_t* bitmapBuffer;
+    uint16_t bitmapWidth;
+    uint16_t bitmapHeight;
+};
+
+struct State {};
+
+using Type = Bmp;
```

Comment:
Any reason this type is declared here, in `xod/icons`? It looks generic enough to be moved in `xod/graphics`.